.github/workflows/vectors-cbor.yml

```
@@ -0,0 +1,35 @@
+name: CBOR Vectors (Python/Go)
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)